packages/trail-core/database/config.js

```diff
@@ -7,7 +7,7 @@ const minimist = require('minimist')
 
 module.exports = function () {
   const argv = minimist(process.argv.slice(2))
-  const version = argv.version
+  const version = '' + argv.version
```

Comment:
Not sure what you're trying to achieve here. If you want version be a string, I'd use `argv.version.toString()`, if you want to make sure you always have a value, I'd use `argv.version || ''`.

A good summary would be: `(argv.version || '').toString()`

Reply:
I wanted to make it a string. Using your latest suggestion will do.